server/models/video/video.ts

```diff
@@ -751,8 +762,20 @@ export class VideoModel extends Model<VideoModel> {
     )
   }
 
-  createThumbnail (videoFile: VideoFileModel) {
-    const imageSize = THUMBNAILS_SIZE.width + 'x' + THUMBNAILS_SIZE.height
+  createThumbnail (videoFile: VideoFileModel, videoRatio: number) {
+    const thumbnailRatio = THUMBNAILS_SIZE.width / THUMBNAILS_SIZE.height
```

Comment:
Please move the height/width calculation in another function so you don't repeat yourself